app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,14 @@
 class PostsController < ApplicationController
   layout "post"
+  include Pundit
```

Comment:
tận dụng kế thừa đi.
Move vào trong application_controller để sử dụng chung cho toàn bộ hệ thống

Reply:
đã fix